src/components/GameView/CardListSortable.vue

```diff
@@ -54,20 +54,10 @@ export default {
       type: String,
       default: 'No Cards Yet',
     },
-    /**
-     * Prefix used to identify where this card is in UI
-     * Used to generate data-* selectors for e2e testing
-     * @example 'scrap-dialog' for this prop
-     * results in labeling cards like
-     *     data-scrap-dialog-card="<suite>-<rank>"
-     */
```

Comment:
Is there any particular reason why you opted to remove this comment? Given that the string is expected to be in a specific format, it seems useful to have an explanation of how it's constructed. I think it's better to leave the comment in, but I'm open to discussion about it

Reply:
No , I removed it in a haste , really sorry I will add all the necessary comments again thanks for the guidance.